Geometry/HcalCommonData/data/hcalforwardshower/v2/hcalforwardshower.xml

```diff
@@ -0,0 +1,311 @@
+  </LogicalPart>
+</LogicalPartSection>
+
+<PosPartSection label="hcalforwardshower.xml">
+  <PosPart copyNumber="1">
+    <rParent name="hcalforwardshower:World"/>
+    <rChild name="hcalforwardshower:HFWorld"/>
+  </PosPart>
+  <PosPart copyNumber="1">
+    <rParent name="hcalforwardshower:HFWorld"/>
+    <rChild name="hcalforwardshower:Chamber"/>
+    <Translation x="0*fm" y="0*fm" z="[chamberZ]"/>
+  </PosPart>
+  <PosPart copyNumber="1">
+    <rParent name="hcalforwardshower:Chamber"/>
+    <rChild name="hcalforwardshower:WirePlane"/>
+    <Translation x="0*fm" y="0*fm" z="[wirePlaneZ]"/>
+  </PosPart>
+  <PosPart copyNumber="1">
+    <rParent name="hcalforwardshower:HFCladL"/>
+    <rChild name="hcalforwardshower:HFFibreL"/>
+  </PosPart>
+  <PosPart copyNumber="2">
+    <rParent name="hcalforwardshower:HFCladS"/>
+    <rChild name="hcalforwardshower:HFFibreS"/>
+  </PosPart>
+  <PosPart copyNumber="5001">
+    <rParent name="hcalforwardshower:HFBox"/>
+    <rChild name="hcalforwardshower:HFCladS"/>
+    <Translation x="-[holeX]" y="-[holeX]" z="[shiftSF]"/>
+  </PosPart>
+  <PosPart copyNumber="5002">
+    <rParent name="hcalforwardshower:HFBox"/>
+    <rChild name="hcalforwardshower:HFCladL"/>
+    <Translation x="-[holeX]" y="[holeX]" z="0*fm"/>
+  </PosPart>
+  <PosPart copyNumber="5003">
+    <rParent name="hcalforwardshower:HFBox"/>
+    <rChild name="hcalforwardshower:HFCladL"/>
+    <Translation x="[holeX]" y="-[holeX]" z="0*fm"/>
+  </PosPart>
+  <PosPart copyNumber="5004">
+    <rParent name="hcalforwardshower:HFBox"/>
+    <rChild name="hcalforwardshower:HFCladS"/>
+    <Translation x="[holeX]" y="[holeX]" z="[shiftSF]"/>
+  </PosPart>
+</PosPartSection>
+
+<Algorithm name="hcal:DDHCalLinearXY">
+  <rParent name="hcalforwardshower:HFWorld"/>
+  <Vector name="Child" type="string" nEntries="1">
+    hcalforwardshower:HFWedge</Vector>
+  <Numeric name="DeltaX" value="2*[wedgeDx]"/>
+  <Numeric name="DeltaY" value="2*[wedgeDy]"/>
+  <Numeric name="NumberX" value="[wedgeX]"/>
+  <Numeric name="NumberY" value="[wedgeY]"/>
+  <Vector name="Center" type="numeric" nEntries="3">
+    0., 0., 182.5*cm
+  </Vector>
+</Algorithm>
+<Algorithm name="hcal:DDHCalLinearXY">
+  <rParent name="hcalforwardshower:HFWedge"/>
+  <Vector name="Child" type="string" nEntries="1">
+    hcalforwardshower:HFBox</Vector>
+  <Numeric name="DeltaX" value="2*[cellDx]"/>
+  <Numeric name="DeltaY" value="2*[cellDy]"/>
+  <Numeric name="NumberX" value="[cellX]"/>
+  <Numeric name="NumberY" value="[cellY]"/>
+  <Vector name="Center" type="numeric" nEntries="3">
+    0., 0., 0.
+  </Vector>
+</Algorithm>
+
+<SpecParSection label="hcalforwardshower.xml" eval="true">
+  <SpecPar name="fibre">
+    <PartSelector path="//HFFibre.*"/>
+    <Parameter name="SensitiveDetector" value="FiberSensitiveDetector" eval="false"/>
+    <Parameter name="ReadOutName" value="FibreHits" eval="false"/>
+    <Parameter name="rTable" value="12.50*cm"/>
```

Comment:
Don't you want the numeric parameters to have `eval="true"`?

Reply:
SpecParSection has eval="true" which is valid for all parts of spectra except some parameters which are string and has eval="false"

Reply:
OK, thanks.